test/integration/appsource/cymbal_bank_test.go

```diff
@@ -0,0 +1,182 @@
+
+	cp "github.com/otiai10/copy"
+)
+
+func TestSourceCymbalBank(t *testing.T) {
+
+	// TODO: switch to an array based on ENVs
+	multitenant := tft.NewTFBlueprintTest(t, tft.WithTFDir("../../../2-multitenant/envs/development"))
+	multitenant_nonprod := tft.NewTFBlueprintTest(t, tft.WithTFDir("../../../2-multitenant/envs/non-production"))
+	multitenant_prod := tft.NewTFBlueprintTest(t, tft.WithTFDir("../../../2-multitenant/envs/production"))
+
+	region := multitenant.GetStringOutputList("cluster_regions")[0]
+	for appName, serviceNames := range testutils.ServicesNames {
+
+		appSourcePath := fmt.Sprintf("../../../6-appsource/%s", appName)
+
+		appfactory := tft.NewTFBlueprintTest(t, tft.WithTFDir(fmt.Sprintf("../../../3-appfactory/apps/%s/frontend/", appName)))
+		projectID := appfactory.GetStringOutput("app_admin_project_id")
+		t.Run(appName, func(t *testing.T) {
+			t.Parallel()
+			for _, serviceName := range serviceNames {
+
+				servicePath := fmt.Sprintf("%s/%s", appSourcePath, serviceName)
+
+				vars := map[string]interface{}{
+					"project_id":                     projectID,
+					"region":                         region,
+					"cluster_membership_id_dev":      multitenant.GetStringOutputList("cluster_membership_ids")[0],
+					"cluster_membership_ids_nonprod": multitenant_nonprod.GetStringOutputList("cluster_membership_ids"),
+					"cluster_membership_ids_prod":    multitenant_prod.GetStringOutputList("cluster_membership_ids"),
+					"buckets_force_destroy":          "true",
+				}
+
+				appsource := tft.NewTFBlueprintTest(t,
+					tft.WithTFDir(servicePath),
+					tft.WithVars(vars),
+					tft.WithRetryableTerraformErrors(testutils.RetryableTransientErrors, 3, 2*time.Minute),
+				)
+
+				appsource.DefineVerify(func(assert *assert.Assertions) {
+
+					appRepo := fmt.Sprintf("https://source.developers.google.com/p/%s/r/eab-%s-%s", projectID, appName, serviceName)
+					prodTarget := "dev"
+
+					// Push cymbal bank app source code
+					tmpDirApp := t.TempDir()
+					gitApp := git.NewCmdConfig(t, git.WithDir(tmpDirApp))
+					gitAppRun := func(args ...string) {
+						_, err := gitApp.RunCmdE(args...)
+						if err != nil {
+							t.Fatal(err)
+						}
+					}
+
+					gitAppRun("clone", "--branch", "v0.6.4", "https://github.com/GoogleCloudPlatform/bank-of-anthos.git", tmpDirApp)
+					gitAppRun("config", "user.email", "eab-robot@example.com")
+					gitAppRun("config", "user.name", "EAB Robot")
+					gitAppRun("config", "--global", "credential.https://source.developers.google.com.helper", "gcloud.sh")
+					gitAppRun("config", "--global", "init.defaultBranch", "main")
+					gitAppRun("config", "--global", "http.postBuffer", "157286400")
+					gitAppRun("checkout", "-b", "main")
+					gitAppRun("remote", "add", "google", appRepo)
+					datefile, err := os.OpenFile(fmt.Sprintf("%s/src/%s/date.txt", tmpDirApp, serviceName), os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
```

Comment:
`Step #32 - "appsource-verify":     cymbal_bank_test.go:94: open /tmp/TestSourceCymbalBankcymbal-bank2689384808/001/src/accounts-contacts/date.txt: no such file or directory`

Hmm, might need some sort of mapping as in the upstream repo it's `src/accounts/contacts`